CHANGELOG.md

```diff
@@ -96,6 +96,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 - [Bug] Check phase name before SearchRequestOperationsListener onPhaseStart ([#12035](https://github.com/opensearch-project/OpenSearch/pull/12035))
 - Fix Span operation names generated from RestActions ([#12005](https://github.com/opensearch-project/OpenSearch/pull/12005))
 - Fix error in RemoteSegmentStoreDirectory when debug logging is enabled ([#12328](https://github.com/opensearch-project/OpenSearch/pull/12328))
+- Update supported version for the primary_only parameter in force-merge API ([#12657](https://github.com/opensearch-project/OpenSearch/pull/12657))
```

Comment:
You do not need another changelog entry for this. Please remote this change and I'll add in the skip changelog label.

Reply:
Thanks, I've removed that, please help to add the `skip-changelog` label.